provider/acsCredential.go

```diff
@@ -11,15 +11,16 @@ import (
 
 // AcsCredential represents the credential file for ACS
 type AcsCredential struct {
-	Cloud             string `json:"cloud"`
-	TenantID          string `json:"tenantId"`
-	SubscriptionID    string `json:"subscriptionId"`
-	ClientID          string `json:"aadClientId"`
-	ClientSecret      string `json:"aadClientSecret"`
-	ResourceGroup     string `json:"resourceGroup"`
-	Region            string `json:"location"`
-	VNetName          string `json:"vnetName"`
-	VNetResourceGroup string `json:"vnetResourceGroup"`
+	Cloud                  string `json:"cloud"`
+	TenantID               string `json:"tenantId"`
+	SubscriptionID         string `json:"subscriptionId"`
+	ClientID               string `json:"aadClientId"`
+	ClientSecret           string `json:"aadClientSecret"`
+	ResourceGroup          string `json:"resourceGroup"`
+	Region                 string `json:"location"`
+	VNetName               string `json:"vnetName"`
+	VNetResourceGroup      string `json:"vnetResourceGroup"`
+	UserAssignedIdentityID string `json:"userAssignedIdentityID"`
```

Comment:
this msi does not have permission to vnet by default, user still need to assign the role manually. @feiskyer